CPP-Example.cpp

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <string.h>
+
+int main()
+{
+   char src[40];
+   char dest[100];
+   char dest2[100];
+
+   gets(src); 
```

Comment:
[Banned C function detected (gets)](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS181021.md)
gets will read in as many bytes as are provided, regardless of the size of buffer recieving the bytes.  This allows for conditions that cause a buffer overlow in the destination buffer.

Suggestions:
**[0]** *+   gets_s(src, <size of src>);*
**[1]** *+   fgets(src, <size of src>, stdin);*